input/docs/editors/vscode.md

```diff
@@ -1,19 +0,0 @@
-Order: 20
```

Comment:
@pascalberger since this page is being removed, can we add a `redirectfrom` on at least one of the other new pages?

Reply:
That's a little bit tricky. `https://cakebuild.net/docs/editors/vscode`  will continue to work, since it now points to the `index.cshtml` in the `vscode` folder. The only thing which will break is `https://cakebuild.net/docs/editors/vscode.html`, which I don't think is used by anybody and also don't think can easily be redirected by Wyam.

Reply:
Makes sense.